src/util/getStripeClient.spec.ts

```diff
@@ -5,10 +5,22 @@ import { getStripeClient } from './getStripeClient';
 describe('getStripeClient', () => {
   const apiKey = 'test';
 
-  describe('when `apiKey` is provided', () => {
-    it('should return the stripe client', () => {
-      const stripeClient = getStripeClient({ apiKey });
-      expect(stripeClient).toBeInstanceOf(Stripe);
+  it('should return the stripe client', () => {
+    const stripeClient = getStripeClient({ apiKey });
+    expect(stripeClient).toBeInstanceOf(Stripe);
+  });
+
+  it('should return the stripe client with customize', () => {
+    const stripeClient = getStripeClient({
+      apiKey,
+      appInfo: {
+        name: 'name',
+        url: 'url',
+        version: 'version',
+      },
+      timeout: 10000,
+      telemetry: false,
```

Comment:
I'm kind of a stickler on this but can you sort this json alphabetically?

Reply:
thank you reviews, fixed it :)